lizmap/modules/lizmap/lib/Request/Proxy.php

```diff
@@ -136,6 +136,35 @@ public static function build($project, $params, $requestXml = null)
         return null;
     }
 
+    /**
+     * Returns the HTTP Status Message.
+     *
+     * @param int $code the HTTP Status Code
+     *
+     * @return string $msg the HTTP Status Message
+     */
+    public static function getHttpStatusMsg($code)
```

Comment:
Perhaps add more error code messages, such as 400, 405 and 500 like in https://github.com/3liz/lizmap-gobsapi-module/blob/master/gobsapi/controllers/apiController.php#L10 ?

Reply:
@mdouchin fixed